.github/workflows/fetch.yaml

```diff
@@ -42,7 +42,7 @@ jobs:
         with:
           add-paths: ./modules/sync/**
           commit-message: "Detected new managed modules references"
-          branch: fetch-versions-${{ github.run_id }}
+          branch: fetch-modules
```

Comment:
See https://github.com/marketplace/actions/create-pull-request#action-behaviour for the behavior of this action without an alternative strategy.

This is how we've been managing the plugins fetch versions workflow for some time.